test/unit/Insurance.js

```diff
@@ -629,26 +630,68 @@ describe("Unit tests: Insurance.sol", function () {
             })
         })
 
-        context("when the leveraged notional value is > 0", async () => {
-            it("returns the appropriate 8 hour funding rate", async () => {
+        context("when the pool is greater than the target", async () => {
```

Comment:
Would you be able to write a test for when the pool is < the target && `(((fund_target - fund_holdings)/fund_target) ** 2) < 1`?

There are tests for the two edge cases (pool above target & pool has no holdings), but just for peace of mind a normal case test would be good.

Reply:
Done